templates/web/index.html

```diff
@@ -15,7 +15,9 @@
     <link rel="stylesheet" href="/assets/fa/css/fontawesome.min.css">
     <link rel="stylesheet" href="/assets/fa/css/regular.min.css">
     <link rel="stylesheet" href="/assets/fa/css/solid.min.css">
+    <link rel="stylesheet" href="https://api.mapbox.com/mapbox-gl-js/v3.8.0/mapbox-gl.css">
 
+    <script src="https://api.mapbox.com/mapbox-gl-js/v3.8.0/mapbox-gl.js"></script>
```

Comment:
Can we pull in the dependencies and bake them into the binary, similar to what I do [here](https://github.com/gbbirkisson/spis/blob/daff7446c7b05f3540bc8576e7b530b14a5344ba/assets/scripts/Makefile). I was very much trying to make an effort to have the server no depend on any external dependencies during runtime.